### ISSUE TITLE
Import from Keepass1 broken?

### ISSUE THREAD

**Issue by ermshiperete** [keepassxreboot/keepassxc#3503]
I have a Keepass1 (.kdb) file that I try to open or import.

## Expected Behavior

Converts the Keepass1 database into a Keepass2 file (.kdbx).

## Current Behavior

It doesn't matter whether I choose Database/Open Database or Database/Import/KeePass 1 database. I get the same result: I get the File Open dialog that allows me to select the database. After entering the password, an error message shows: "Error while reading the database: The selected file is an old KeePass 1 database (.kdb). You can import it by clicking on Database > 'Import KeePass 1 database...'", and the Enter password dialog stays...

## Debug Info
KeePassXC - Version 2.4.3
Revision: 5d6ef0c

Qt 5.9.5
Debugging mode is disabled.

Operating system: Ubuntu 18.04.3 LTS
CPU architecture: x86_64
Kernel: linux 5.0.0-25-generic

Enabled extensions:
- Auto-Type
- Browser Integration
- SSH Agent
- KeeShare (signed and unsigned sharing)
- YubiKey

Cryptographic libraries:
 libgcrypt 1.8.1

**Comment by droidmonkey** [comment]
I cannot replicate at all using Database -> Import KeePass1

**Comment by ermshiperete** [comment]
Did you try on Linux? Seems it works on Windows, but on Linux I always get the bug.

**Comment by droidmonkey** [comment]
Whaaaaaa?? I'll try this evening

**Comment by chloesoe** [comment]
Same problem here. 
```
KeePassXC - Version 2.4.3
Revision: 5d6ef0c

Qt 5.9.5
Debugging mode is disabled.

Operating system: Linux Mint 19.2
CPU architecture: x86_64
Kernel: linux 4.15.0-62-generic

Enabled extensions:
- Auto-Type
- Browser Integration
- SSH Agent
- KeeShare (signed and unsigned sharing)
- YubiKey

Cryptographic libraries:
 libgcrypt 1.8.1
```

**Comment by droidmonkey** [comment]
I cannot replicate this using the 2.4.3 snap on Ubuntu.

**Comment by ermshiperete** [comment]
Very strange. I installed the snap version 2.4.3 on Ubuntu. With that I was able to import two different KeePass 1 databases. Then I tried again with version 2.4.3 installed from the official Ubuntu ppa. With that I was able to open one of the KeePass 1 databases, but with the other (which is the one I originally tried) it still fails. Strange. I wonder what the difference is.

Anyways, since there is a workaround (use snap package or Windows for the import) I'm fine with this issue being closed.
